packages/python/accern_data/accern_data.py

```diff
@@ -633,7 +636,8 @@ def download_range(
                     filters=valid_filters,
                     progress_bar=progress_bar)
         progress_bar.close()
-        self._expected_records = []
+        if len(self._error_list) > 0:
```

Comment:
Only if it was an actual error. If we suppress errors we don't print them here. The user has to check